src/main/java/uk/gov/companieshouse/search/api/mapper/ElasticSearchResponseMapper.java

```diff
@@ -183,6 +185,10 @@ public Company mapEnhancedSearchResponse(SearchHit hit) {
             enhancedCompany.setDateOfCreation(LocalDate.parse((String) currentCompanyMap.get(DATE_OF_CREATION), enhancedFormatter));
         }
 
+        if (currentCompanyMap.containsKey(SIC_CODES_KEY)) {
```

Comment:
A test case for this would be good to bring the PR's coverage up higher.

Reply:
sorted